### DISCUSSION TITLE
Ultrawide Primary Monitor

### DISCUSSION THREAD

**Discussion by ddulic** [LizardByte/community#69] [Sunshine]
Hello, is anyone successfully using Sunshine with an Ultrawide Primary Monitor?

As mentioned at the bottom of https://github.com/LizardByte/Sunshine/issues/176#issuecomment-1216369368, my PD is 3440x1440p (43:18 aspect ratio) and I have to scale the resolution manually before I stream to avoid black borders on the casting 4k TV (21:9 aspect ratio).

Does anyone have a solution?

Thanks!

**Comment by mirh** [comment]
I mean, what else could you do?
If the source is wider than the target, you either pan and scan (aka crop) or you letterbox (aka black borders). 
Also, I'm pretty sure that 4K tvs are 16:9.

**Comment by vhenzor** [comment]
If on Windows, have a look at [Gamestream_launchpad](https://github.com/cgarst/gamestream_launchpad) or [QRes](https://www.majorgeeks.com/files/details/qres.html) to automatically change your resolution and screen refresh rate as discussed on this reddit topic : https://www.reddit.com/r/ShieldAndroidTV/comments/x9abmn/automatically_cap_the_fps_to_60/

[a comment by an automated account is not shown]
